specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/schema/binding/test.schema.patch.json

```diff
@@ -70,6 +70,52 @@
                                         }
                                     }
                                 ]
+                            },
+                            "catalogs":
+                            {
+                                "type": "array",
+                                "items":
+                                {
+                                    "type": "string"
+                                }
+                            },
+                            "guards":
+                            {
+                                "type": "array",
+                                "items":
+                                {
+                                    "type": "object",
+                                    "properties":
+                                    {
+                                        "guard":
+                                        {
+                                            "type": "string"
+                                        },
+                                        "token":
+                                        {
+                                            "type": "string"
+                                        }
+                                    }
+                                }
+                            },
```

Comment:
Please change to `authorization` with `patternProperties` and `credentials` as described in other feedback comment.